src/_includes/v2/footer/socials.html

```diff
@@ -0,0 +1,5 @@
+<p>
+  <a class="font-weight-bold" href="mailto:hello@calitp.org">hello@calitp.org</a>
+</p>
```

Comment:
This is trivial, but it's not really a paragraph and I'm allergic to extraneous tags when a little CSS will do :)


```suggestion
<a class="d-block mt-4 mb-3 font-weight-bold" href="mailto:hello@calitp.org">hello@calitp.org</a>
```

(it's not highlighted in the diff, but note the added classes)

Reply:
my instinct was to avoid any additional CSS, but since these are all canned bootstrap helpers, i think your suggestion is better.